### ISSUE TITLE
"clickable" span lacks proper interactive element semantics

### ISSUE THREAD

**Issue by savitris** [frameless/developer.overheid.nl#81]
**Dit probleem kan op meerdere pagina's aanwezig zijn, waaronder:**
[CBS Cijfers Open Data](https://apis.developer.overheid.nl/apis/cbs-cijfers)

**Probleembeschrijving:**
Het label (naast de Header 1) in een span wordt door de screenreader aangekondigd als 'klikbaar', maar het is geen interactief element.

**Referentiecode:**
`<span class="pill pill--white hide-upto-sm mb-1 ml-2">OData</span>`

**Gerelateerde WCAG:**
WCAG 4.1.2 Name, Role, Value

**Referentieafbeelding:**

![A screenreader when focused on a span with the text OData says the element is clickable](https://github.com/user-attachments/assets/10dc9cf5-1c28-440d-8fa3-a509dd7c9486)

_**Referentienummer:** A089_
